app/views/characters/_skill.html.erb

```diff
@@ -0,0 +1,7 @@
+<% skills.each do |skill| %>
+    <option value="<%= skill.name %>"><%= skill.name %></option>
```

Comment:
Is this representing a single skill or a group of skills? I think the partial name does not make a lot of sense